docs/client.rst

```diff
@@ -52,5 +58,43 @@ Example script
     ## Test the client that it can still send and recieve data from the nodes after the remap has been done
     print(rc.set('foo', 'bar'))
 
+This feature is also useful in cases such as when one is trying to access AWS ElastiCache cluster secured by Stunnel (https://www.stunnel.org/)
+
+.. code-block:: python
 
-Pleaes note that this host_port_remap feature will not work on the startup_nodes so you still need to put in a valid and reachable set of startup nodes.
+    from rediscluster import RedisCluster
+
+    startup_nodes = [
+        {"host": "127.0.0.1", "port": "17000"},
+        {"host": "127.0.0.1", "port": "17001"},
+        {"host": "127.0.0.1", "port": "17002"},
+        {"host": "127.0.0.1", "port": "17003"},
+        {"host": "127.0.0.1", "port": "17004"},
+        {"host": "127.0.0.1", "port": "17005"}
+    ]
+
+    host_port_remap=[
+        {'from_host': '41.1.3.1', 'from_port': 6379, 'to_host': '127.0.0.1', 'to_port': 17000},
+        {'from_host': '41.1.3.5', 'from_port': 6379, 'to_host': '127.0.0.1', 'to_port': 17001},
+        {'from_host': '41.1.4.2', 'from_port': 6379, 'to_host': '127.0.0.1', 'to_port': 17002},
+        {'from_host': '50.0.1.7', 'from_port': 6379, 'to_host': '127.0.0.1', 'to_port': 17003},
+        {'from_host': '50.0.7.3', 'from_port': 6379, 'to_host': '127.0.0.1', 'to_port': 17004},
+        {'from_host': '32.0.1.1', 'from_port': 6379, 'to_host': '127.0.0.1', 'to_port': 17005}
+    ]
+
+
+    # Note: decode_responses must be set to True when used with python3
+    rc = RedisCluster(
+        startup_nodes=startup_nodes,
+        host_port_remap=host_port_remap,
+        decode_responses=True,
+        ssl=True,
+        ssl_cert_reqs=None,
+        # Needed for Elasticache Clusters
+        skip_full_coverage_check=True)
+
+
+    print(rc.connection_pool.nodes.nodes)
```

Comment:
Note: I have not included comments here cause they are they same as the previous example